src/components/pages/VolunteersPage/VolunteersPage.js

```diff
@@ -124,7 +124,10 @@ class VolunteersPage extends React.Component {
               <>
                 <h2>{volunteerGroup.letter}</h2>
                 <hr />
-                <CardColumns style={{ paddingBottom: '1em' }}>
+                <CardColumns
+                  key={volunteerGroup.letter}
```

Comment:
Not sure that this belongs here - key should be on the parent (see above comment).

Reply:
I see